fastchat/conversation.py

```diff
@@ -896,6 +896,19 @@ def get_conv_template(name: str) -> Conversation:
     )
 )
 
+# WizardCoder default template, same as alpaca
```

Comment:
1. Could you put a link to the source here?
2. If it is the same as alpaca, we may not need to create a new template for it, we can just return "alpaca" in `model_adapter.py`

Reply:
Thank you for your feedback, I have changed the implementation to return the alpaca template directly from the ModelAdapter, without having to register a new conversation template and add the link in comment. Does it ok for you? Thanks